packages/esm-form-entry-app/src/app/services/monthly-scheduled-resource.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { MonthlyScheduleParams } from '../types';
+import { WindowRef } from '../window-ref';
+
+@Injectable()
+export class MonthlyScheduleResourceService {
+  constructor(private http: HttpClient, private windowRef: WindowRef) {}
+
+  public getMonthlySchedule(params: MonthlyScheduleParams) {
+    const etlUrl = `/etl-latest/etl/get-monthly-schedule?startDate=${params.startDate}&endDate=${params.endDate}&locationUuids=${params.locationUuids}&limit=${params.limit}&programType=${params.programType}&groupBy=groupByPerson,groupByAttendedDate,groupByRtcDate`;
```

Comment:
@donaldkibet this line remains a problem (I think). This is specific to ampath and needs to be configuration. I will also file a jira ticket to faciliate creation of a similar endpoint based on the bahmni appointments data model.

Reply:
@jonathandick i have made the etl filed configurable